README.md

```
@@ -51,6 +52,97 @@
+We follow **GitFlow** branching strategy to maintain code quality and streamline development.  
+
+<img src="public/images/GitFlow-Branching-Strategy.png" alt="Gitflow Diagram" width="100%">
+
+---
+
+## 📜 API Documentation  
+We provide a fully interactive **Swagger UI** for API testing and documentation.  
+
+![Swagger](path/to/swagger-ui.png)  
+
+🔗 [View API Documentation](#)  
```

markdownlint-cli2 (0.17.2)

No empty links
null

(MD042, no-empty-links)